dist/module/locale.js

```diff
@@ -319,7 +319,7 @@ export var COUNTRY_LANGS = {
   GT: [LANG.ES, LANG.EN, LANG.FR, LANG.ZH],
   GW: [LANG.EN, LANG.FR, LANG.ES, LANG.ZH],
   GY: [LANG.EN, LANG.FR, LANG.ES, LANG.ZH],
-  HK: [LANG.EN, LANG.ZH],
+  HK: [LANG.EN, LANG.ZH, LANG.ZH_HANT],
```

Comment:
Add `LANG.ZH_HANT` to `TW` as well.  Taiwan only uses Traditional Chinese and has the same issue.

Reply:
Added!